### PR TITLE
[WNMGDS-314, 313, 310] Update backstop docs and reference files

### PR DESCRIPTION
**Pull request by bernardwang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,22 +6,22 @@
...
-| [Support](packages/support/) | Sass variables, mixins, and functions. Included in the Core package. <br> [![@cmsgov/design-system-support](https://img.shields.io/npm/v/@cmsgov/design-system-support.svg?label=@cmsgov%2Fdesign-system-support)](https://www.npmjs.com/package/@cmsgov/design-system-support) |
+| Name                         | Description                                                                                                                                                                                                                                                                           |
+| ---------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```
Comment (line47) [inline README.md]: The spacing here seems strange, is this right?

File: README.md [hunk 2]
```diff
@@ -6,22 +6,22 @@
...
+| [ESLint config](packages/eslint-config-design-system/)       | The ESLint rules we use to lint the design system's JS and React components <br> [![@cmsgov/eslint-config-design-system](https://img.shields.io/npm/v/@cmsgov/eslint-config-design-system.svg?label=@cmsgov%2Feslint-config-design-system)](https://www.npmjs.com/package/@cmsgov/eslint-config-design-system) |
+| [Stylelint config](packages/stylelint-config-design-system/) | The Stylelint rules we use to lint the design system's Sass <br> [![@cmsgov/stylelint-config-design-system](https://img.shields.io/npm/v/@cmsgov/stylelint-config-design-system.svg?label=@cmsgov%2Fstylelint-config-design-system)](https://www.npmjs.com/package/@cmsgov/stylelint-config-design-system)     |
+| [Yeoman generator](packages/generator-cmsgov/)               | A [Yeoman](http://yeoman.io/) generator used in the development process. Again, unless you're a contributor, this directory isn't that interesting to you.                                                                                                                                                     |
```
Comment (line47) [inline README.md]: we should update `backstop.json` to remove the select page as that is no longer live on the site. We should also remove the screenshots for that page as well..

File: README.md [hunk 3]
```diff
@@ -47,56 +47,56 @@ _Note_: When you create a Git commit, any staged scripts will be automatically r
...
+- `yarn test:e2e`
+  - Runs end to end tests
+- `yarn test:e2e packages/core/Autocomplete`
+  - Runs a single end to end test, this example runs Autocomplete
+- `yarn test:watch`
+  - Runs JS unit tests and will continue to run tests as files change
+- `yarn update-snapshots`
+  - Updates [Jest snapshots](http://facebook.github.io/jest/docs/en/snapshot-testing.html)
+- `yarn lint`
+  - Runs just the linting portion of the tests
```
Comment (line47) [inline README.md]: There are a few images that are broken 😢 

- backstop_default_CMSDS_-_Components_-_Select_0_document_1_tablet.png
- backstop_default_CMSDS_-_Utilities_-_Clearfix_0_document_0_phone.png
- backstop_default_CMSDS_-_Utilities_-_Clearfix_0_document_1_tablet
- backstop_default_CMSDS_-_Utilities_-_Clearfix_0_document_2_desktop
- backstop_default_CMSDS_-_Utilities_-_Truncate_0_document_0_phone
- backstop_default_CMSDS_-_Utilities_-_Truncate_0_document_1_tablet
- backstop_default_CMSDS_-_Utilities_-_Truncate_0_document_2_desktop

File: README.md (hunk 1 above)
Comment (bernardwang) [reply README.md]: yeah. im not sure what happened here...

Comment (bernardwang) [reply README.md]: Actually it looks like thats `prettier`'s default formatting. I think it's making the table the same dimension for each row.

**Comment by bernardwang** [comment]
@line47 Feedback should be addressed

**Comment by line47** [comment]
I noticed some font changes on the USA banner link text at the top of the page and also with the responsive UI panel. Could we hide that entire responsive panel when we run the scans? 

ezgif com-video-to-gif

**Comment by bernardwang** [comment]
@line47  are the font changes failing certain tests?

**Comment by line47** [comment]
I was getting 38 fails when running backstop test from this branch

**Comment by bernardwang** [comment]
@line47 The most recent update should fix this!

**Pull request merged**